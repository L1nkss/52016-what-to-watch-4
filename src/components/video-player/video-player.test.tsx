import * as renderer from "react-test-renderer";
import VideoPlayer from "./video-player";

const mockOne = {
  poster: `img/bohemian-rhapsody.jpg`,
  src: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  videoRef: React.createRef(),
  setupRef: () => {}
};

describe(`Testing VideoPlayer component`, () => {
  it(`Component should successfully rendered`, () => {
    const component = renderer.create(<VideoPlayer videoRef={mockOne.videoRef} setupRef={mockOne.setupRef} poster={mockOne.poster} src={mockOne.src} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
