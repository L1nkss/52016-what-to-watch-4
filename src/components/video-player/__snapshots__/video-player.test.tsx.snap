// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing VideoPlayer component Component should successfully rendered 1`] = `
<video
  className="player__video"
  muted={true}
  poster="img/bohemian-rhapsody.jpg"
/>
`;
